config/locales/en.yml

```diff
@@ -110,16 +110,16 @@ en:
         one_or_two_choices: "Only one box must be ticked or 'other disabilities' plus one of mobility disabilities"
       prevten:
         non_temp_accommodation: "Answer cannot be non-temporary accommodation as you already told us this is a re-let to tenant who occupied the same property as temporary accommodation"
+      referral:
+        secure_tenancy: "Answer must be internal transfer as you already told us this is a secure tenancy"
+        rsnvac_non_temp: "Answer cannot be this source of referral as you already told us this is a re-let to tenant who occupied the same property as temporary accommodation"
```

Comment:
@@dan13L3 do these error messages work given that they can answer these in either order?

Reply:
@baarkerlounger Would you or one of the other devs be able to run these screens with me on a call so I can make sure I've understood the full context?

Reply:
Yup, happy to do that. Will set up some time with you

Reply:
@baarkerlounger Reviewed this with @kosiakkatrina and should be good to go now